code/datums/status_effects/debuffs.dm

```diff
@@ -23,14 +23,25 @@
 	owner.adjustFireLoss(0.1)
 	owner.adjustToxLoss(0.2)
 
-/datum/status_effect/cultghost //is a cult ghost and can't use manifest runes
+/datum/status_effect/cultghost //is a cult ghost and can't use manifest runes, can see ghosts and dies if too far from summoner
 	id = "cult_ghost"
 	duration = -1
 	alert_type = null
+	var/damage = 7.5
+	var/source_UID
+
+/datum/status_effect/cultghost/on_creation(mob/living/new_owner, mob/living/source)
+	. = ..()
+	source_UID = source.UID()
 
 /datum/status_effect/cultghost/tick()
 	if(owner.reagents)
 		owner.reagents.del_reagent("holywater") //can't be deconverted
+	var/mob/living/summoner = locateUID(source_UID)
+	if(summoner in range(owner, 21))
+		return
+	owner.adjustBruteLoss(damage)
+	to_chat(owner, "<span class='warning'>You are too far away from the summoner!</span>")
```

Comment:
```suggestion
	to_chat(owner, "<span class='userdanger'>You are too far away from the summoner!</span>")
```

Might be worth upping this span's intensity since this might kill em fast